components/api/src/test/java/com/hotels/styx/api/io/ResourceFactoryTest.java

```diff
@@ -26,7 +26,7 @@
     public void canAcquireClasspathResources() {
         Resource resource = ResourceFactory.newResource("classpath:com/hotels/styx/api/io/resource.txt");
         
-        assertThat(resource, contains("This is an example resource.\nIt has content to use in automated tests."));
+        assertThat(resource, contains("This is an example resource."+System.lineSeparator()+"It has content to use in automated tests."));
```

Comment:
We may want to consider using `String.format` (with static imports) for these line separator cases.

e.g. ` assertThat(resource, contains(format("This is an example resource.%nIt has content to use in automated tests.")));`